src/matrix/matrixCenterZMean.ts

```diff
@@ -1,10 +1,10 @@
 import { ArrayType } from '..';
 
-/**
+/**.
```

Comment:
Did you add this manually? The dot should be at the end of the description, not here.
The `eslint-fix` script (or the vscode-eslint extension if you have set it up to fix on save) should correct it automatically.

Reply:
When i save this point appear

Reply:
@lpatiny have you seen this happen before?

Reply:
I can reproduce, this is an annoying issue...

Reply:
It is a bug in the ESLint plugin: https://github.com/gajus/eslint-plugin-jsdoc/issues/779
I suggest that you search for `/**.` in the project and manually move the dot to the end of the sentence.

Reply:
okey

Reply:
When i delete it manually => Warning : "Sentence must end with a period."
and when I save the file the point reappears

Reply:
You should not only delete it, but also add a dot at the end of the sentence (i.e move the dot)

Reply:
done

Reply:
i got the same warning